$OpenBSD: patch-tasks_spec_rake,v 1.2 2012/04/06 19:28:22 jeremy Exp $

The rcov.opts file is not shipped with the gem, and isn't
necessary as we aren't running the rcov task.

--- tasks/spec.rake.orig	Thu Apr  5 08:46:41 2012
+++ tasks/spec.rake	Thu Apr  5 08:46:46 2012
@@ -7,5 +7,4 @@ end
 RSpec::Core::RakeTask.new(:rcov) do |rcov|
   rcov.pattern    = "./spec/**/*_spec.rb"
   rcov.rcov       = true
-  rcov.rcov_opts  = File.read('spec/rcov.opts').split(/\s+/)
 end
